www/views/confirm.html

```diff
@@ -7,7 +7,7 @@
     </ion-nav-back-button>
   </ion-nav-bar>
 
-  <ion-content ng-class="{'slide-to-pay': !hasClick && !insuffientFunds && !noMatchingWallet}">
```

Comment:
Agree, but it 's not working at first launch (after clicking on a bitcoin: URI). We should tweak that function in another PR.